google/ggrc-core
# GGRC-6588 Send flags on event update

# Issue description

Task Assignee is able to send invite to another user after calendar event was modified

# Steps to test the changes

Steps tp reproduce:
1. Create any WF
2. Create a task
3. Activate WF
4. Run a cron job to see the event in google calendar
5. Modify task: change description > Run a cron job once again
6. Edit event in calendar and send invite to another user: success
Actual Result: Task Assignee is able to send invite to another user after CT was modified
Expected Result: Task Assignee should not be able to send invite to another user after CT was modified

# Solution description

Send custom values for `sendNotifications`,  `guestsCanModify`,  `guestsCanInviteOthers` parameters in Calendar API

Use `locked` option
https://developers.google.com/calendar/v3/reference/events

# Sanity checklist




# PR Review checklist
